frontend/utility/platform-osx.mm

```diff
@@ -15,20 +15,15 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ******************************************************************************/
 
-#include <sstream>
-#include <dlfcn.h>
-#include <util/base.h>
-#include <util/threading.h>
-#include <obs-config.h>
-#include "platform.hpp"
-#include "obs-app.hpp"
+#import "platform.hpp"
 
-#include <unistd.h>
+#import <OBSApp.hpp>
+
+#import <util/threading.h>
 
-#import <AppKit/AppKit.h>
-#import <CoreFoundation/CoreFoundation.h>
 #import <AVFoundation/AVFoundation.h>
-#import <ApplicationServices/ApplicationServices.h>
+#import <AppKit/AppKit.h>
```

Comment:
In case-insensitive order, `AppKit` comes before `AVFoundation`. Not sure what the intent here is.

Reply:
`AV` comes before `Ap` because capital letters always rank higher than lowercase letters.